app/controllers/admin/dashboard_controller.rb

```diff
@@ -2,6 +2,7 @@ class Admin::DashboardController < Admin::BaseController
 
   def index
     @cohorts = Cohort.all.to_json(:include => [:applications, :reviewers])
+    @user = jwt_encode({user: current_user.to_json})
```

Comment:
I would recommend encoding just the `id` and then looking it up on the server. It reduces the amount of data we're passing around and keeps things simpler until we need them to be more comlicated.

Reply:
I agree. Easy fix.

Reply:
Token is now just the User ID.